### PR TITLE
Add installment CITI support

### PR DESCRIPTION
**Pull request by ratchagarn**
## Purpose

Add Installment CITI support

## Description

- Added Installment CITI support to payment methods

## Quality assurance

Make sure Installment CITI can be selected and sources can be created.

## Operations impact

N/A

## Pre- and post-deployment steps

N/A

## Business impact (release notes)

N/A

## Add labels and assign reviewers

N/A

## Back-out Procedure

N/A

### CODE REVIEW

**Review by nuxzero — APPROVED** [review]

File: sdk/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -73,6 +73,7 @@
     <string name="payment_method_internet_banking_bbl_title">バンコク銀行</string>
     <string name="payment_method_internet_banking_ktb_title">クルンタイ銀行</string>
     <string name="payment_method_internet_banking_scb_title">サイアム・コマーシャル銀行</string>
+    <string name="payment_method_internet_banking_citi_title">シティバンク銀行</string>
```
Comment (nlogx) [inline sdk/src/main/res/values-ja/strings.xml:76]: `payment_method_internet_banking_citi_title`? Not payment_method_installment_citi_title?

File: sdk/src/sharedTest/java/co/omise/android/ui/InstallmentChooserFragmentTest.kt [hunk 2]
```diff
@@ -25,6 +25,7 @@ class InstallmentChooserFragmentTest {
             PaymentMethod(name = "installment_kbank"),
             PaymentMethod(name = "installment_ktc"),
             PaymentMethod(name = "installment_scb")
+            PaymentMethod(name = "installment_citi")
```
Comment (nlogx) [inline sdk/src/sharedTest/java/co/omise/android/ui/InstallmentChooserFragmentTest.kt:28]: Missing comma after `PaymentMethod(name = "installment_scb")`

Comment (nuxzero) [reply sdk/src/sharedTest/java/co/omise/android/ui/InstallmentChooserFragmentTest.kt:28]: Good eyes 👍  👁️

**Comment by nuxzero** [comment]
This PR is no longer updated.

**Pull request closed without merging**